lib/rules/no-unused-prop-types.js

```diff
@@ -63,7 +56,7 @@ module.exports = {
     var configuration = Object.assign({}, defaults, context.options[0] || {});
     var skipShapeProps = configuration.skipShapeProps;
     var customValidators = configuration.customValidators || [];
-    var propWrapperFunctions = new Set(configuration.propWrapperFunctions || []);
+    var propWrapperFunctions = new Set(context.settings.propWrapperFunctions || []);
```

Comment:
Yes, I tested both with and without settings defined - `context.settings` will always be an object.